### PR TITLE
Migrate bump version.py script from wazuh packages

### PR DESCRIPTION
**Pull request by lchico**
|Related issue|
|---|
|#23953|

## Description
This PR migrates the `bump_version.py` script to a Bash version and introduces a feature for updating release dates. The bump_version.sh script is updated with the functionality required from the original Python script ([link to bump_version.py](https://github.com/wazuh/wazuh-packages/blob/master/bump_version.py)), along with the addition of an option to update the release date.

### How to use

image

## Tests
Using the bump_version.py script I got:
<details>
<summary>Python script output:  </summary>

```bash
# Date: 10 Jun 2024
python3 ../tools/bump_version.py -v 4.10.0
git diff package > python_script.diff
```

```diff
diff --git a/packages/debs/SPECS/wazuh-agent/debian/changelog b/packages/debs/SPECS/wazuh-agent/debian/changelog
index aadf3c6586..65ea2e57bb 100644
--- a/packages/debs/SPECS/wazuh-agent/debian/changelog
+++ b/packages/debs/SPECS/wazuh-agent/debian/changelog
@@ -1,3 +1,9 @@
+wazuh-agent (4.10.0-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/release-4-10-0.html
+
+ -- Wazuh, Inc <info@wazuh.com>  Mon, 10 Jun 2024 00:00:00 +0000
+
 wazuh-agent (4.9.0-RELEASE) stable; urgency=low
 
   * More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
diff --git a/packages/debs/SPECS/wazuh-agent/debian/copyright b/packages/debs/SPECS/wazuh-agent/debian/copyright
index 9d0f205188..a66319a8d3 100644
--- a/packages/debs/SPECS/wazuh-agent/debian/copyright
+++ b/packages/debs/SPECS/wazuh-agent/debian/copyright
@@ -1,6 +1,6 @@
 This work was packaged for Debian by:
 
-    Wazuh, Inc <info@wazuh.com> on Tue, 14 May 2024 00:00:00 +0000
+    Wazuh, Inc <info@wazuh.com> on Mon, 10 Jun 2024 00:00:00 +0000
 
 It was downloaded from:
 
diff --git a/packages/debs/SPECS/wazuh-manager/debian/changelog b/packages/debs/SPECS/wazuh-manager/debian/changelog
index c72a0434eb..1f97a3d259 100644
--- a/packages/debs/SPECS/wazuh-manager/debian/changelog
+++ b/packages/debs/SPECS/wazuh-manager/debian/changelog
@@ -1,3 +1,9 @@
+wazuh-manager (4.10.0-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/release-4-10-0.html
+
+ -- Wazuh, Inc <info@wazuh.com>  Mon, 10 Jun 2024 00:00:00 +0000
+
 wazuh-manager (4.9.0-RELEASE) stable; urgency=low
 
   * More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
diff --git a/packages/debs/SPECS/wazuh-manager/debian/copyright b/packages/debs/SPECS/wazuh-manager/debian/copyright
index 9d0f205188..a66319a8d3 100644
--- a/packages/debs/SPECS/wazuh-manager/debian/copyright
+++ b/packages/debs/SPECS/wazuh-manager/debian/copyright
@@ -1,6 +1,6 @@
 This work was packaged for Debian by:
 
-    Wazuh, Inc <info@wazuh.com> on Tue, 14 May 2024 00:00:00 +0000
+    Wazuh, Inc <info@wazuh.com> on Mon, 10 Jun 2024 00:00:00 +0000
 
 It was downloaded from:
 
diff --git a/packages/macos/specs/wazuh-agent-arm64.pkgproj b/packages/macos/specs/wazuh-agent-arm64.pkgproj
index 9c463b1a19..c06df83893 100644
--- a/packages/macos/specs/wazuh-agent-arm64.pkgproj
+++ b/packages/macos/specs/wazuh-agent-arm64.pkgproj
@@ -812,7 +812,7 @@
 				<key>USE_HFS+_COMPRESSION</key>
 				<false/>
 				<key>VERSION</key>
-				<string>4.9.0-1</string>
+				<string>4.10.0-1</string>
 			</dict>
 			<key>TYPE</key>
 			<integer>0</integer>
@@ -1240,7 +1240,7 @@
 				</dict>
 			</array>
 			<key>NAME</key>
-			<string>wazuh-agent-4.9.0-1.arm64</string>
+			<string>wazuh-agent-4.10.0-1.arm64</string>
 			<key>PAYLOAD_ONLY</key>
 			<false/>
 			<key>TREAT_MISSING_PRESENTATION_DOCUMENTS_AS_WARNING</key>
diff --git a/packages/macos/specs/wazuh-agent-intel64.pkgproj b/packages/macos/specs/wazuh-agent-intel64.pkgproj
index 743fd71890..271c372a43 100644
--- a/packages/macos/specs/wazuh-agent-intel64.pkgproj
+++ b/packages/macos/specs/wazuh-agent-intel64.pkgproj
@@ -812,7 +812,7 @@
 				<key>USE_HFS+_COMPRESSION</key>
 				<false/>
 				<key>VERSION</key>
-				<string>4.9.0-1</string>
+				<string>4.10.0-1</string>
 			</dict>
 			<key>TYPE</key>
 			<integer>0</integer>
@@ -1239,7 +1239,7 @@
 				</dict>
 			</array>
 			<key>NAME</key>
-			<string>wazuh-agent-4.9.0-1.intel64</string>
+			<string>wazuh-agent-4.10.0-1.intel64</string>
 			<key>PAYLOAD_ONLY</key>
 			<false/>
 			<key>TREAT_MISSING_PRESENTATION_DOCUMENTS_AS_WARNING</key>
diff --git a/packages/rpms/SPECS/wazuh-agent.spec b/packages/rpms/SPECS/wazuh-agent.spec
index 5ccbeac5f4..4a4dccb577 100644
--- a/packages/rpms/SPECS/wazuh-agent.spec
+++ b/packages/rpms/SPECS/wazuh-agent.spec
@@ -678,6 +678,8 @@ rm -fr %{buildroot}
 %attr(750, root, wazuh) %{_localstatedir}/wodles/gcloud/*
 
 %changelog
+* Mon Jun 10 2024 support <info@wazuh.com> - 4.10.0
+- More info: https://documentation.wazuh.com/current/release-notes/release-4-10-0.html
 * Tue May 14 2024 support <info@wazuh.com> - 4.9.0
 - More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
 * Tue Mar 26 2024 support <info@wazuh.com> - 4.8.2
diff --git a/packages/rpms/SPECS/wazuh-manager.spec b/packages/rpms/SPECS/wazuh-manager.spec
index 56e66f2f9d..fa4b9aeeb5 100644
--- a/packages/rpms/SPECS/wazuh-manager.spec
+++ b/packages/rpms/SPECS/wazuh-manager.spec
@@ -895,6 +895,8 @@ rm -fr %{buildroot}
 %attr(750, root, wazuh) %{_localstatedir}/wodles/gcloud/*
 
 %changelog
+* Mon Jun 10 2024 support <info@wazuh.com> - 4.10.0
+- More info: https://documentation.wazuh.com/current/release-notes/release-4-10-0.html
 * Tue May 14 2024 support <info@wazuh.com> - 4.9.0
 - More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
 * Thu Apr 25 2024 support <info@wazuh.com> - 4.7.4
```

</details>

<details>
<summary>Bash script output:  </summary>

```bash
# Date: 12 Jun 2024
./bump_version.sh -r 1 -v v4.10.0
```

```diff
diff --git a/packages/debs/SPECS/wazuh-agent/debian/changelog b/packages/debs/SPECS/wazuh-agent/debian/changelog
index aadf3c6586..a97afbcc6e 100644
--- a/packages/debs/SPECS/wazuh-agent/debian/changelog
+++ b/packages/debs/SPECS/wazuh-agent/debian/changelog
@@ -1,3 +1,9 @@
+wazuh-agent (4.10.0-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/release-4-10-0.html
+
+ -- Wazuh, Inc <info@wazuh.com>  Wed, 12 Jun 2024 00:00:00 +0000
+
 wazuh-agent (4.9.0-RELEASE) stable; urgency=low
 
   * More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
diff --git a/packages/debs/SPECS/wazuh-agent/debian/copyright b/packages/debs/SPECS/wazuh-agent/debian/copyright
index 9d0f205188..e8acc4748d 100644
--- a/packages/debs/SPECS/wazuh-agent/debian/copyright
+++ b/packages/debs/SPECS/wazuh-agent/debian/copyright
@@ -1,6 +1,6 @@
 This work was packaged for Debian by:
 
-    Wazuh, Inc <info@wazuh.com> on Tue, 14 May 2024 00:00:00 +0000
+    Wazuh, Inc <info@wazuh.com> on Wed, 12 Jun 2024 00:00:00 +0000
 
 It was downloaded from:
 
diff --git a/packages/debs/SPECS/wazuh-manager/debian/changelog b/packages/debs/SPECS/wazuh-manager/debian/changelog
index c72a0434eb..8ef6e32bec 100644
--- a/packages/debs/SPECS/wazuh-manager/debian/changelog
+++ b/packages/debs/SPECS/wazuh-manager/debian/changelog
@@ -1,3 +1,9 @@
+wazuh-manager (4.10.0-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/release-4-10-0.html
+
+ -- Wazuh, Inc <info@wazuh.com>  Wed, 12 Jun 2024 00:00:00 +0000
+
 wazuh-manager (4.9.0-RELEASE) stable; urgency=low
 
   * More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
diff --git a/packages/debs/SPECS/wazuh-manager/debian/copyright b/packages/debs/SPECS/wazuh-manager/debian/copyright
index 9d0f205188..e8acc4748d 100644
--- a/packages/debs/SPECS/wazuh-manager/debian/copyright
+++ b/packages/debs/SPECS/wazuh-manager/debian/copyright
@@ -1,6 +1,6 @@
 This work was packaged for Debian by:
 
-    Wazuh, Inc <info@wazuh.com> on Tue, 14 May 2024 00:00:00 +0000
+    Wazuh, Inc <info@wazuh.com> on Wed, 12 Jun 2024 00:00:00 +0000
 
 It was downloaded from:
 
diff --git a/packages/macos/specs/wazuh-agent-arm64.pkgproj b/packages/macos/specs/wazuh-agent-arm64.pkgproj
index 9c463b1a19..c06df83893 100644
--- a/packages/macos/specs/wazuh-agent-arm64.pkgproj
+++ b/packages/macos/specs/wazuh-agent-arm64.pkgproj
@@ -812,7 +812,7 @@
 				<key>USE_HFS+_COMPRESSION</key>
 				<false/>
 				<key>VERSION</key>
-				<string>4.9.0-1</string>
+				<string>4.10.0-1</string>
 			</dict>
 			<key>TYPE</key>
 			<integer>0</integer>
@@ -1240,7 +1240,7 @@
 				</dict>
 			</array>
 			<key>NAME</key>
-			<string>wazuh-agent-4.9.0-1.arm64</string>
+			<string>wazuh-agent-4.10.0-1.arm64</string>
 			<key>PAYLOAD_ONLY</key>
 			<false/>
 			<key>TREAT_MISSING_PRESENTATION_DOCUMENTS_AS_WARNING</key>
diff --git a/packages/macos/specs/wazuh-agent-intel64.pkgproj b/packages/macos/specs/wazuh-agent-intel64.pkgproj
index 743fd71890..271c372a43 100644
--- a/packages/macos/specs/wazuh-agent-intel64.pkgproj
+++ b/packages/macos/specs/wazuh-agent-intel64.pkgproj
@@ -812,7 +812,7 @@
 				<key>USE_HFS+_COMPRESSION</key>
 				<false/>
 				<key>VERSION</key>
-				<string>4.9.0-1</string>
+				<string>4.10.0-1</string>
 			</dict>
 			<key>TYPE</key>
 			<integer>0</integer>
@@ -1239,7 +1239,7 @@
 				</dict>
 			</array>
 			<key>NAME</key>
-			<string>wazuh-agent-4.9.0-1.intel64</string>
+			<string>wazuh-agent-4.10.0-1.intel64</string>
 			<key>PAYLOAD_ONLY</key>
 			<false/>
 			<key>TREAT_MISSING_PRESENTATION_DOCUMENTS_AS_WARNING</key>
diff --git a/packages/rpms/SPECS/wazuh-agent.spec b/packages/rpms/SPECS/wazuh-agent.spec
index 5ccbeac5f4..0cd5699165 100644
--- a/packages/rpms/SPECS/wazuh-agent.spec
+++ b/packages/rpms/SPECS/wazuh-agent.spec
@@ -678,6 +678,8 @@ rm -fr %{buildroot}
 %attr(750, root, wazuh) %{_localstatedir}/wodles/gcloud/*
 
 %changelog
+* Wed Jun 12 2024 support <info@wazuh.com> - 4.10.0
+- More info: https://documentation.wazuh.com/current/release-notes/release-4-10-0.html
 * Tue May 14 2024 support <info@wazuh.com> - 4.9.0
 - More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
 * Tue Mar 26 2024 support <info@wazuh.com> - 4.8.2
diff --git a/packages/rpms/SPECS/wazuh-manager.spec b/packages/rpms/SPECS/wazuh-manager.spec
index 56e66f2f9d..cad368184b 100644
--- a/packages/rpms/SPECS/wazuh-manager.spec
+++ b/packages/rpms/SPECS/wazuh-manager.spec
@@ -895,6 +895,8 @@ rm -fr %{buildroot}
 %attr(750, root, wazuh) %{_localstatedir}/wodles/gcloud/*
 
 %changelog
+* Wed Jun 12 2024 support <info@wazuh.com> - 4.10.0
+- More info: https://documentation.wazuh.com/current/release-notes/release-4-10-0.html
 * Tue May 14 2024 support <info@wazuh.com> - 4.9.0
 - More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
 * Thu Apr 25 2024 support <info@wazuh.com> - 4.7.4
```
</details>

<details>
<summary>Diff output between python and bas script: </summary>
The only difference here is the date bump date.

`diff bash_script.diff python_script.py`
```diff
2c2
< index aadf3c6586..a97afbcc6e 100644
---
> index aadf3c6586..65ea2e57bb 100644
10c10
< + -- Wazuh, Inc <info@wazuh.com>  Wed, 12 Jun 2024 00:00:00 +0000
---
> + -- Wazuh, Inc <info@wazuh.com>  Mon, 10 Jun 2024 00:00:00 +0000
16c16
< index 9d0f205188..e8acc4748d 100644
---
> index 9d0f205188..a66319a8d3 100644
23c23
< +    Wazuh, Inc <info@wazuh.com> on Wed, 12 Jun 2024 00:00:00 +0000
---
> +    Wazuh, Inc <info@wazuh.com> on Mon, 10 Jun 2024 00:00:00 +0000
28c28
< index c72a0434eb..8ef6e32bec 100644
---
> index c72a0434eb..1f97a3d259 100644
36c36
< + -- Wazuh, Inc <info@wazuh.com>  Wed, 12 Jun 2024 00:00:00 +0000
---
> + -- Wazuh, Inc <info@wazuh.com>  Mon, 10 Jun 2024 00:00:00 +0000
42c42
< index 9d0f205188..e8acc4748d 100644
---
> index 9d0f205188..a66319a8d3 100644
49c49
< +    Wazuh, Inc <info@wazuh.com> on Wed, 12 Jun 2024 00:00:00 +0000
---
> +    Wazuh, Inc <info@wazuh.com> on Mon, 10 Jun 2024 00:00:00 +0000
98c98
< index 5ccbeac5f4..0cd5699165 100644
---
> index 5ccbeac5f4..4a4dccb577 100644
105c105
< +* Wed Jun 12 2024 support <info@wazuh.com> - 4.10.0
---
> +* Mon Jun 10 2024 support <info@wazuh.com> - 4.10.0
111c111
< index 56e66f2f9d..cad368184b 100644
---
> index 56e66f2f9d..fa4b9aeeb5 100644
118c118
< +* Wed Jun 12 2024 support <info@wazuh.com> - 4.10.0
---
> +* Mon Jun 10 2024 support <info@wazuh.com> - 4.10.0
```
</details>

<details>
<summary>Using update release date: </summary>

```bash
./bump_version.sh -d 2024-06-10 
git diff package > bash_script_update_bump_date.diff
```
</details>

<details>
<summary> bash_script_update_bump_date.diff </summary>

```
diff --git a/packages/debs/SPECS/wazuh-agent/debian/changelog b/packages/debs/SPECS/wazuh-agent/debian/changelog
index aadf3c6586..65ea2e57bb 100644
--- a/packages/debs/SPECS/wazuh-agent/debian/changelog
+++ b/packages/debs/SPECS/wazuh-agent/debian/changelog
@@ -1,3 +1,9 @@
+wazuh-agent (4.10.0-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/release-4-10-0.html
+
+ -- Wazuh, Inc <info@wazuh.com>  Mon, 10 Jun 2024 00:00:00 +0000
+
 wazuh-agent (4.9.0-RELEASE) stable; urgency=low
 
   * More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
diff --git a/packages/debs/SPECS/wazuh-agent/debian/copyright b/packages/debs/SPECS/wazuh-agent/debian/copyright
index 9d0f205188..a66319a8d3 100644
--- a/packages/debs/SPECS/wazuh-agent/debian/copyright
+++ b/packages/debs/SPECS/wazuh-agent/debian/copyright
@@ -1,6 +1,6 @@
 This work was packaged for Debian by:
 
-    Wazuh, Inc <info@wazuh.com> on Tue, 14 May 2024 00:00:00 +0000
+    Wazuh, Inc <info@wazuh.com> on Mon, 10 Jun 2024 00:00:00 +0000
 
 It was downloaded from:
 
diff --git a/packages/debs/SPECS/wazuh-manager/debian/changelog b/packages/debs/SPECS/wazuh-manager/debian/changelog
index c72a0434eb..1f97a3d259 100644
--- a/packages/debs/SPECS/wazuh-manager/debian/changelog
+++ b/packages/debs/SPECS/wazuh-manager/debian/changelog
@@ -1,3 +1,9 @@
+wazuh-manager (4.10.0-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/release-4-10-0.html
+
+ -- Wazuh, Inc <info@wazuh.com>  Mon, 10 Jun 2024 00:00:00 +0000
+
 wazuh-manager (4.9.0-RELEASE) stable; urgency=low
 
   * More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
diff --git a/packages/debs/SPECS/wazuh-manager/debian/copyright b/packages/debs/SPECS/wazuh-manager/debian/copyright
index 9d0f205188..a66319a8d3 100644
--- a/packages/debs/SPECS/wazuh-manager/debian/copyright
+++ b/packages/debs/SPECS/wazuh-manager/debian/copyright
@@ -1,6 +1,6 @@
 This work was packaged for Debian by:
 
-    Wazuh, Inc <info@wazuh.com> on Tue, 14 May 2024 00:00:00 +0000
+    Wazuh, Inc <info@wazuh.com> on Mon, 10 Jun 2024 00:00:00 +0000
 
 It was downloaded from:
 
diff --git a/packages/macos/specs/wazuh-agent-arm64.pkgproj b/packages/macos/specs/wazuh-agent-arm64.pkgproj
index 9c463b1a19..c06df83893 100644
--- a/packages/macos/specs/wazuh-agent-arm64.pkgproj
+++ b/packages/macos/specs/wazuh-agent-arm64.pkgproj
@@ -812,7 +812,7 @@
                                <key>USE_HFS+_COMPRESSION</key>
                                <false/>
                                <key>VERSION</key>
-                               <string>4.9.0-1</string>
+                               <string>4.10.0-1</string>
                        </dict>
                        <key>TYPE</key>
                        <integer>0</integer>
@@ -1240,7 +1240,7 @@
                                </dict>
                        </array>
                        <key>NAME</key>
-                       <string>wazuh-agent-4.9.0-1.arm64</string>
+                       <string>wazuh-agent-4.10.0-1.arm64</string>
                        <key>PAYLOAD_ONLY</key>
                        <false/>
                        <key>TREAT_MISSING_PRESENTATION_DOCUMENTS_AS_WARNING</key>
diff --git a/packages/macos/specs/wazuh-agent-intel64.pkgproj b/packages/macos/specs/wazuh-agent-intel64.pkgproj
index 743fd71890..271c372a43 100644
--- a/packages/macos/specs/wazuh-agent-intel64.pkgproj
+++ b/packages/macos/specs/wazuh-agent-intel64.pkgproj
@@ -812,7 +812,7 @@
                                <key>USE_HFS+_COMPRESSION</key>
                                <false/>
                                <key>VERSION</key>
-                               <string>4.9.0-1</string>
+                               <string>4.10.0-1</string>
                        </dict>
                        <key>TYPE</key>
                        <integer>0</integer>
@@ -1239,7 +1239,7 @@
                                </dict>
                        </array>
                        <key>NAME</key>
-                       <string>wazuh-agent-4.9.0-1.intel64</string>
+                       <string>wazuh-agent-4.10.0-1.intel64</string>
                        <key>PAYLOAD_ONLY</key>
                        <false/>
                        <key>TREAT_MISSING_PRESENTATION_DOCUMENTS_AS_WARNING</key>
diff --git a/packages/rpms/SPECS/wazuh-agent.spec b/packages/rpms/SPECS/wazuh-agent.spec
index 5ccbeac5f4..4a4dccb577 100644
--- a/packages/rpms/SPECS/wazuh-agent.spec
+++ b/packages/rpms/SPECS/wazuh-agent.spec
@@ -678,6 +678,8 @@ rm -fr %{buildroot}
 %attr(750, root, wazuh) %{_localstatedir}/wodles/gcloud/*
 
 %changelog
+* Mon Jun 10 2024 support <info@wazuh.com> - 4.10.0
+- More info: https://documentation.wazuh.com/current/release-notes/release-4-10-0.html
 * Tue May 14 2024 support <info@wazuh.com> - 4.9.0
 - More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
 * Tue Mar 26 2024 support <info@wazuh.com> - 4.8.2
diff --git a/packages/rpms/SPECS/wazuh-manager.spec b/packages/rpms/SPECS/wazuh-manager.spec
index 56e66f2f9d..fa4b9aeeb5 100644
--- a/packages/rpms/SPECS/wazuh-manager.spec
+++ b/packages/rpms/SPECS/wazuh-manager.spec
@@ -895,6 +895,8 @@ rm -fr %{buildroot}
 %attr(750, root, wazuh) %{_localstatedir}/wodles/gcloud/*
 
 %changelog
+* Mon Jun 10 2024 support <info@wazuh.com> - 4.10.0
+- More info: https://documentation.wazuh.com/current/release-notes/release-4-10-0.html
 * Tue May 14 2024 support <info@wazuh.com> - 4.9.0
 - More info: https://documentation.wazuh.com/current/release-notes/release-4-9-0.html
 * Thu Apr 25 2024 support <info@wazuh.com> - 4.7.4
```
</details>

**Note:** The `diff bash_script_update_bump_date.diff python_script.diff` is **empty**.

### CODE REVIEW

**Review by mjcr99 — APPROVED** [review]
LGTM, for the rest of reviewers, I checked the changes were properly done comparing with this PR wazuh/wazuh-packages#2982

File: tools/bump_version.sh [hunk 1]
```diff
@@ -217,3 +262,60 @@ then
 
     sed -E -i'' -e "s/^(VIProductVersion \").+\"/\1$product\"/g" $NSIS_FILE
 fi
+
+# Wazuh Packages
+if [ -z "$version" ]
+then
+    UPDATE_RELESE_DATE="yes"
```
Comment (MarcelKemp) [inline tools/bump_version.sh:269]: Let's fix the following typo:
```suggestion
    UPDATE_RELEASE_DATE="yes"
```

File: tools/bump_version.sh [hunk 2]
```diff
@@ -217,3 +262,60 @@ then
...
+    echo "Use default date: $bump_date"
+fi
+
+bump_date=$(LC_ALL=en_US.UTF-8 TZ="UTC" date -d "$bump_date" +"%a, %d %b %Y %H:%M:%S %z")
+
+# SPECS files
+spec_date=$(LC_ALL=en_US.UTF-8 TZ="UTC" date -d "$bump_date" +"%a %b %d %Y")
+for spec_file in $SPEC_FILES; do
+    echo "Bumping version in $spec_file"
+    if [ -z "$UPDATE_RELESE_DATE" ] ; then
```
Comment (MarcelKemp) [inline tools/bump_version.sh:290]:
```suggestion
    if [ -z "$UPDATE_RELEASE_DATE" ] ; then
```

File: tools/bump_version.sh [hunk 3]
```diff
@@ -217,3 +262,60 @@ then
...
+    else
+       sed -E -i'' "/%changelog/{N;s/\n.*support/\n* $spec_date support/}" $spec_file
+    fi
+done
+
+# Deb changelog files
+for changelog_file in $CHANGELOG_FILES; do
+    echo "Bumping version in $changelog_file"
+    install_type=$(sed -E 's/.*wazuh-(manager|agent).*/wazuh-\1/' <<< $changelog_file)
+    if [ -z "$UPDATE_RELESE_DATE" ] ; then
```
Comment (MarcelKemp) [inline tools/bump_version.sh:302]:
```suggestion
    if [ -z "$UPDATE_RELEASE_DATE" ] ; then
```

File: tools/bump_version.sh [hunk 4]
```diff
@@ -217,3 +262,60 @@ then
...
+    bump_date=$(LC_ALL=en_US.UTF-8 TZ="UTC" date +"%F")
+    echo "Use default date: $bump_date"
+fi
+
+bump_date=$(LC_ALL=en_US.UTF-8 TZ="UTC" date -d "$bump_date" +"%a, %d %b %Y %H:%M:%S %z")
+
+# SPECS files
+spec_date=$(LC_ALL=en_US.UTF-8 TZ="UTC" date -d "$bump_date" +"%a %b %d %Y")
+for spec_file in $SPEC_FILES; do
+    echo "Bumping version in $spec_file"
```
Comment (MarcelKemp) [inline tools/bump_version.sh:289]: As in the case of _Packages_ there is no version bump, let's change the text to: 
```suggestion
    echo "Updating the release date of $version in $spec_file"
```

File: tools/bump_version.sh [hunk 5]
```diff
@@ -217,3 +262,60 @@ then
...
+        sed -E -i'' "/%changelog/a * $spec_date support <info@wazuh.com> - ${VERSION}\n\
+- More info: https://documentation.wazuh.com/current/release-notes/release-$mayor-$minor-$patch.html" $spec_file
+    else
+       sed -E -i'' "/%changelog/{N;s/\n.*support/\n* $spec_date support/}" $spec_file
+    fi
+done
+
+# Deb changelog files
+for changelog_file in $CHANGELOG_FILES; do
+    echo "Bumping version in $changelog_file"
```
Comment (MarcelKemp) [inline tools/bump_version.sh:300]:
```suggestion
    echo "Updating the release date of $version in $spec_file"
```

**Review by MarcelKemp — CHANGES_REQUESTED** [review]
GJ, just a couple of minor changes.

File: tools/bump_version.sh [hunk 6]
```diff
@@ -34,42 +88,33 @@ do
 
         shift 2
         ;;
-    *)
-        if [[ $1 =~ ^v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+(-[[:alnum:]]+)?$ ]]
+    "-d")
+        if [[ $2 =~ ^[0-9]{4}-((0[1-9])|(1[0-2]))-(([0-2][1-9])|(3[01]))$ ]]
```
Comment (MarcelKemp) [inline tools/bump_version.sh:92]: The following change is needed for days: `10` and `20`.
```suggestion
        if [[ $2 =~ ^[0-9]{4}-((0[1-9])|(1[0-2]))-(([0-2][0-9])|(3[01]))$ ]]
```

**Review by MarcelKemp — CHANGES_REQUESTED** [review]

File: tools/bump_version.sh (hunk 6 above)
Comment (lchico) [reply tools/bump_version.sh]: Fixed! Thanks!

**Review by MarcelKemp — APPROVED** [review]

**Comment by MarcelKemp** [comment]
## Failing checks

### Unit tests: cmocka server
- https://github.com/wazuh/wazuh-jenkins/issues/6554

### Build Linux x86_64/arm64v8 manager rpm package
- Both are expected, because _wazuh-packages_ from `4.9.0` onwards are not being updated, causing these checks to fail due to lack of SPECS updates.

**Pull request merged**